docs/02.API-REFERENCE.md

```diff
@@ -175,6 +175,21 @@ Native free callback of an object
 typedef void (*jerry_object_free_callback_t) (const uintptr_t native_p);
 ```
 
+## jerry_object_native_info_t
+
+**Summary**
+
+The type infomation of the native handle. It includes the free callback of the native handle.
```

Comment:
native handle => native pointer

"It includes the free callback that will be called when associated JavaScript object is garbage collected. It can be left `NULL` in case it is not needed."